core/src/main/java/GameObjects/URLBuilder.java

```diff
@@ -0,0 +1,22 @@
+package GameObjects;
+
+/**
+ * The UrlBuilder class is a utility class used to build the URL for OpenDB required to retrieve
+ * questions given preferences regarding number of questions, category, and difficulty.
+ */
+public final class UrlBuilder implements UrlBuilderWrapper {
```

Comment:
This should be called "UrlBuilderImpl" since it implements UrlBuilder interface.